src/main/java/fi/fmi/avi/model/taf/TAF.java

```diff
@@ -23,4 +32,10 @@
 
     Optional<TAFReference> getReferredReport();
 
+    boolean isCancelMessage();
```

Comment:
As result of discussion on this subject, after all it may be safer to retain `isCancelMessage` as a true property in order to be able to parse invalid messages having corresponding IWXXM attribute value `true` but missing `cancelledReportValidPeriod`.

`isMissingMessage` ([comment below](https://github.com/fmidev/fmi-avi-messageconverter/pull/68#discussion_r523937535)) is a related but different case.

Reply:
cancelMessage property retained as a separate property